client/src/containers/overview/project-details/parameters/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { atom, useAtom } from "jotai";
+import {
+  PROJECT_SIZE_FILTER,
+  COST_TYPE_SELECTOR,
+  PROJECT_PRICE_TYPE,
+} from "@shared/entities/projects.entity";
+import { z } from "zod";
+
+import {
+  FILTER_KEYS,
+  INITIAL_ABATEMENT_POTENTIAL_RANGE,
+  INITIAL_COST_RANGE,
+} from "@/app/(overview)/constants";
+import { filtersSchema } from "@/app/(overview)/url-store";
+
+import { Label } from "@/components/ui/label";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+const INITIAL_FILTERS_STATE: z.infer<typeof filtersSchema> = {
+  keyword: "",
+  projectSizeFilter: PROJECT_SIZE_FILTER.MEDIUM,
+  priceType: PROJECT_PRICE_TYPE.OPEN_BREAK_EVEN_PRICE,
+  costRangeSelector: COST_TYPE_SELECTOR.NPV,
+  countryCode: "",
+  ecosystem: [],
+  activity: [],
+  activitySubtype: [],
+  costRange: INITIAL_COST_RANGE,
+  abatementPotentialRange: INITIAL_ABATEMENT_POTENTIAL_RANGE,
+};
```

Comment:
We are using only 3 properties here: `projectSizeFilter`, `priceType` and `costRange`. Remove the rest of them.